### DISCUSSION TITLE
Task 10 problem

### DISCUSSION THREAD

**Discussion by burneldog** [DRMPN/IML-and-AML#2] [Q&A] [answered]
I'm sorry, but I don't understand the error. 
I need help :(
![image](https://github.com/DRMPN/IML-and-AML/assets/136190338/e81b045b-0b17-4a53-94c2-c5769c5c8880)

**Comment by DRMPN** [comment, marked as answer]
The pictures are stored in a folder named 'test'.
Copy the pictures from the folder and replace the 'cat image'.
Alternatively, you can upload them as it is easier.
**initial:**
![image](https://github.com/DRMPN/IML-and-AML/assets/61294398/9e4cfba0-0417-4e2f-bf6b-0e27af00fe4b)
**before:**
![image](https://github.com/DRMPN/IML-and-AML/assets/61294398/27f21a81-777a-4cca-aeb4-8ceb6043f2a5)
**after:**
![image](https://github.com/DRMPN/IML-and-AML/assets/61294398/a08e9a99-0ac8-4be5-b77e-d510fed90945)
Following the same process, upload the desired images.

**Reply by burneldog** [reply]
It worked! yields probability, thanks <3
